src/config.json

```diff
@@ -0,0 +1 @@
+{"secret":"encrypt","algorithm":"AES"}
```

Comment:
algorithm value not used?

Reply:
Yes. For now, created the config method and just used the secret. But this solution may not work. Since the config is based on a file on each config set the file will get overridden. So need to think of some alternate solution to maintain the config object instead of keeping the values in a file.

DO NOT MERGE